consensus/obcpbft/obc-batch.go

```diff
@@ -310,6 +316,11 @@ func (op *obcBatch) processMessage(ocMsg *pb.Message, senderHandle *pb.PeerID) e
 	}
 
 	if req := batchMsg.GetRequest(); req != nil {
+		if !op.deduplicator.IsNew(req) {
```

Comment:
My concern is that execution and request reception is not serialized the same way on all replicas, so some might reject the request while others will accept it.  I guess as long as the primary is attaching the request again, we should be fine.